web/templates/web/garden/_navigation.html

```diff
@@ -0,0 +1,15 @@
+{% load web_extras %}
+<div class="row py-lg-3 justify-content-evenly align-items-center">
+    <div class="col-12 col-lg-10">
+        <ul class="nav nav-tabs justify-content-center">
+            {% nav_link request 'web:garden-home' 'Project GARDEN' list=True %}
+            {% nav_link request 'web:garden-about' 'About' list=True %}
+            {% nav_link request 'web:garden-participate' 'How to Join' list=True %}
+            <li class="nav-item">
+                <a class="nav-link navbar-link link-secondary text-center px-3"
+                   href="{{ base_url }}{{ garden_url }}">Start Now</a>
+            </li>
```

Comment:
@okaycj do you know if it's possible to pass the link to the GARDEN lab site (`/studies/garden`) into the `nav_link` template tag here? I tried but couldn't figure it out :frowning_face:

Reply:
You have an example of what I would try down below. 
```
{% url "web:lab-studies-list" lab_slug="garden" as garden_url %}
```

Reply:
Ah, so I forgot to mention that I did try exactly what you suggest here, with the nav link like this:
```
{% nav_link request garden_url 'Start Now' list=True %}
```
But that gave me this error when trying to load any page that uses the garden navigation: 
> Reverse for '/studies/garden/' not found. '/studies/garden/' is not a valid view function or pattern name.

The reason is that the `url` template tag returns the absolute path, i.e. `/studies/garden`, but the `nav_link` template tag expects the url argument to be in the form of a name for reverse lookup.

I just now realized that instead of trying to get the GARDEN studies page in the right form for reverse lookup, I could just edit the `nav_link` template tag so that it allows the option of receiving an absolute path. Does that seem like a reasonable solution to you?